debian/ralf.init.d

```diff
@@ -153,6 +153,11 @@ do_start()
         #   1 if daemon was already running
         #   2 if daemon could not be started
 
+        # Exit if there is no ralf hostname configured
```

Comment:
This should probably be in main.cpp with the verification of the other parameters.